### ISSUE TITLE
[docdb] Endpoint /rpcz should print timeout and elapsed time in same units

### ISSUE THREAD

**Issue by bmatican** [yugabyte/yugabyte-db#2801]
Example
```
{
                    "header": {
                        "call_id": 6794412,
                        "remote_method": {
                            "service_name": "yb.cdc.CDCService",
                            "method_name": "GetChanges"
                        },
                        "timeout_millis": 30000
                    },
                    "micros_elapsed": 7595479,
                    "state": "SENT"
                }
```
cc @spolitov

**Comment by srivignessh** [comment]
@bmatican Can you close the issue? Thanks.

**Comment by bmatican** [comment]
Fixed in 87962cc4fcd37dbed5f515dcff6a54651fbc7c45
